mesheryctl/Makefile

```diff
@@ -67,7 +67,10 @@ docs:
 
 
 e2e-libs:
-	@ cd tests/e2e/$(BATS_LIB_PATH); \
+	@ if [ ! -d "tests/e2e/$(BATS_LIB_PATH)" ]; then \
+		mkdir -p tests/e2e/$(BATS_LIB_PATH); \
+	fi; \
```

Comment:
curious if we can simplify to mkdir -p and remove the if condition as the -p flag will not error if it exists.